### PR TITLE
add curl to check cluster health

### PR DESCRIPTION
**Pull request by spgreenberg**
## Changes proposed in this pull request:

Adds:
- missing content type header to PUT to reenable allocation
- curl to check cluster health 

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/BRANCH_NAME)

## Security Considerations

None

### CODE REVIEW

File: _docs/ops/runbook/troubleshooting-logsearch.md [hunk 1]
```diff
@@ -20,13 +20,19 @@ curl localhost:9200/_cluster/settings?pretty
...
 
 ### Check cluster health
-Log into any `elasticsearch_*` node and [check the cluster health]((#check-cluster-health">}}). If the cluster state is not green or has a high number of pending tasks then ElasticSearch cannot accept data from the ingestors.  Resolve the issues with the ElasticSearch cluster and once its state is green, the ingestor queues should be begin to drain.
+Log into any `elasticsearch_*` node and check the cluster health:
+
+```
+curl localhost:9200/_cluster/health
+```
+
+If the cluster state is not green or has a high number of pending tasks then ElasticSearch cannot accept data from the ingestors.  Resolve the issues with the ElasticSearch cluster and once its state is green, the ingestor queues should be begin to drain.
```
Comment (ccostino) [inline _docs/ops/runbook/troubleshooting-logsearch.md:35]: Do we have any notion of what the potential issues might be and any documentation that exists to help troubleshoot them?  Or would that be a separate PR and/or meant for somewhere else?

File: _docs/ops/runbook/troubleshooting-logsearch.md [hunk 2]
```diff
@@ -20,13 +20,19 @@ curl localhost:9200/_cluster/settings?pretty
...
 ```sh
-curl -X PUT localhost:9200/_cluster/settings -d '{"transient":{"cluster.routing.allocation.enable":"all"}}'
+curl -X PUT localhost:9200/_cluster/settings -H "Content-Type: application/json" -d '{"transient":{"cluster.routing.allocation.enable":"all"}}'
 ```
 
 If reenabling allocation doesn't restore the cluster, [manually reassigning unallocated shards may be necessary]((#force-reallocation %})
 
 ### Check cluster health
-Log into any `elasticsearch_*` node and [check the cluster health]((#check-cluster-health">}}). If the cluster state is not green or has a high number of pending tasks then ElasticSearch cannot accept data from the ingestors.  Resolve the issues with the ElasticSearch cluster and once its state is green, the ingestor queues should be begin to drain.
+Log into any `elasticsearch_*` node and check the cluster health:
```
Comment (ccostino) [inline _docs/ops/runbook/troubleshooting-logsearch.md]: This is likely a basic question, but do we have any instructions/documentation on how to login to the Elasticsearch nodes?  Does it follow an existing pattern we have established elsewhere (such as just logging into VMs via BOSH), or is it unique to anything else?

**Review by ccostino — COMMENTED** [review]
Thank you, @spgreenberg!  A couple of questions came to mind as I went through these updates.  If the questions are better answered elsewhere or as a follow up in the future that's fine!

File: _docs/ops/runbook/troubleshooting-logsearch.md (hunk 2 above)
Comment (spgreenberg) [reply _docs/ops/runbook/troubleshooting-logsearch.md]: It is the same process as any other bosh deployed VM. It might be worth changing this to `bosh ssh elasticsearch_*` to be explicit.

File: _docs/ops/runbook/troubleshooting-logsearch.md (hunk 1 above)
Comment (spgreenberg) [reply _docs/ops/runbook/troubleshooting-logsearch.md:35]: I don't know of any other docs. I made these changes as I tried to use the guide.

File: _docs/ops/runbook/troubleshooting-logsearch.md (hunk 2 above)
Comment (ccostino) [reply _docs/ops/runbook/troubleshooting-logsearch.md]: Sounds good, thank you!

**Review by ccostino — APPROVED** [review]
Thanks, @spgreenberg!
